StitchCoreTestUtils-iOS/StitchCoreTestUtils-iOS/BaseStitchIntTestCocoaTouch.swift

```diff
@@ -0,0 +1,77 @@
+import Foundation
+import StitchCoreTestUtils
+import StitchCore
+import StitchCore_iOS
+import StitchCoreAdminClient
+import XCTest
+
+open class BaseStitchIntTestCocoaTouch: BaseStitchIntTest {
+    var clients = [StitchAppClient]()
+    
+    override open func setUp() {
+        super.setUp()
+        
+        do { try Stitch.initialize() }
+        catch { XCTFail(error.localizedDescription) }
+    }
+    
+    override open func tearDown() {
+        super.tearDown()
+        clients.forEach { $0.auth.logout { _ in } }
+    }
+    
+    override open var stitchBaseURL: String {
```

Comment:
[minor] This needs to be made configurable by passing something to properties or the test.